### PR TITLE
Refactor terminology: Updated 'Parameters' and 'Args' to 'Arguments' for consistency across the codebase.

### PR DESCRIPTION
**Pull request by DevyManish**
This PR resolves inconsistencies in the terminology within function and method docstrings across the project, addressing Issue **[#1968 :  [DOCS] Consistency in function and method docstrings](https://github.com/Avaiga/taipy/issues/1968#top)** . The following updates have been made:

- Replaced occurrences of 'parameter' with 'argument' for improved clarity.
- Renamed 'Parameters' and 'Args' sections to 'Arguments' for uniformity.
- Standardized the use of 'args' to 'arguments' throughout the documentation.

These changes enhance the readability and consistency of the documentation, making it more intuitive for both contributors and users to navigate and understand.

@FlorianJacta @jrobinAV Please take a look at this PR.

### CODE REVIEW

File: taipy/common/config/config.py [hunk 1]
```diff
@@ -283,8 +283,8 @@ def _register(cls, section) -> None:
                 sections[section.id] = section
         else:
             cls._python_config._sections[section.name] = {section.id: section}
-        cls._serializer._section_class[section.name] = section.__class__
-        cls.__json_serializer._section_class[section.name] = section.__class__
+            cls._serializer._section_class[section.name] = section.__class__
+            cls.__json_serializer._section_class[section.name] = section.__class__
```
Comment (jrobinAV) [inline taipy/common/config/config.py]:
```suggestion
        cls._serializer._section_class[section.name] = section.__class__
        cls.__json_serializer._section_class[section.name] = section.__class__
```

File: taipy/_run.py [hunk 2]
```diff
@@ -30,11 +30,11 @@ def _run(*services: _AppType, **kwargs) -> t.Optional[Flask]:
 
     A Taipy service is an instance of a class that runs code as a Web application.
 
-    Parameters:
+    Arguments:
         *services (Union[`Gui^`, `Rest^`, `Orchestrator^`]): Services to run, as separate arguments.<br/>
             If several services are provided, all the services run simultaneously.<br/>
             If this is empty or set to None, this method does nothing.
-        **kwargs: Other parameters to provide to the services.
+        **kwargs: Other Arguments to provide to the services.
```
Comment (jrobinAV) [inline taipy/_run.py]:
```suggestion
        **kwargs: Other parameters to provide to the services.
```

File: taipy/common/config/config.pyi [hunk 3]
```diff
@@ -496,20 +496,20 @@ class Config:
...
+        Arguments:
             id (str): The unique identifier of the new Parquet data node configuration.
             default_path (Optional[str]): The default path of the Parquet file.
             engine (Optional[str]): Parquet library to use. Possible values are *"fastparquet"* or
                 *"pyarrow"*.<br/>
                 The default value is *"pyarrow"*.
             compression (Optional[str]): Name of the compression to use. Possible values are *"snappy"*,
                 *"gzip"*, *"brotli"*, or *"none"* (no compression). The default value is *"snappy"*.
-            read_kwargs (Optional[dict]): Additional parameters passed to the `pandas.read_parquet()`
+            read_kwargs (Optional[dict]): Additional arguments passed to the `pandas.read_parquet()`
```
Comment (jrobinAV) [inline taipy/common/config/config.pyi]:
```suggestion
            read_kwargs (Optional[dict]): Additional parameters passed to the `pandas.read_parquet()`
```

File: taipy/common/config/config.pyi [hunk 4]
```diff
@@ -496,20 +496,20 @@ class Config:
...
                 function.
-            write_kwargs (Optional[dict]): Additional parameters passed to the
+            write_kwargs (Optional[dict]): Additional arguments passed to the
```
Comment (jrobinAV) [inline taipy/common/config/config.pyi]:
```suggestion
            write_kwargs (Optional[dict]): Additional parameters passed to the
```

File: taipy/common/config/config.pyi [hunk 5]
```diff
@@ -496,20 +496,20 @@ class Config:
...
                 `pandas.DataFrame.write_parquet()` function.<br/>
-                The parameters in *read_kwargs* and *write_kwargs* have a **higher precedence** than the
-                top-level parameters which are also passed to Pandas.
+                The arguments in *read_kwargs* and *write_kwargs* have a **higher precedence** than the
+                top-level arguments which are also passed to Pandas.
```
Comment (jrobinAV) [inline taipy/common/config/config.pyi]:
```suggestion
                The parameters in *read_kwargs* and *write_kwargs* have a **higher precedence** 
                than the top-level parameters which are also passed to Pandas.
```

File: taipy/common/config/config.pyi [hunk 6]
```diff
@@ -610,12 +610,12 @@ class Config:
...
         """Configure a new *in-memory* data node configuration.
 
-        Parameters:
+        Arguments:
             id (str): The unique identifier of the new in_memory data node configuration.
             default_data (Optional[any]): The default data of the data nodes instantiated from
                 this in_memory data node configuration.
                 If provided, note that the default_data will be stored as a configuration attribute.
-                So it is designed to handle small data values like parameters, and it must be Json serializable.
+                So it is designed to handle small data values like arguments, and it must be Json serializable.
```
Comment (jrobinAV) [inline taipy/common/config/config.pyi]:
```suggestion
                So it is designed to handle small data values like parameters, and it must be Json serializable.
```

File: taipy/common/config/config.pyi [hunk 7]
```diff
@@ -642,13 +642,13 @@ class Config:
...
 
-        Parameters:
+        Arguments:
             id (str): The unique identifier of the new pickle data node configuration.
             default_path (Optional[str]): The path of the pickle file.
             default_data (Optional[any]): The default data of the data nodes instantiated from
                 this pickle data node configuration.
                 If provided, note that the default_data will be stored as a configuration attribute.
-                So it is designed to handle small data values like parameters, and it must be Json serializable.
+                So it is designed to handle small data values like arguments, and it must be Json serializable.
```
Comment (jrobinAV) [inline taipy/common/config/config.pyi]:
```suggestion
                So it is designed to handle small data values like parameters, and it must be Json serializable.
```

File: taipy/core/common/_mongo_connector.py [hunk 8]
```diff
@@ -20,9 +20,9 @@ def _connect_mongodb(
 ) -> pymongo.MongoClient:
     """Create a connection to a Mongo database.
     The `"mongodb_extra_args"` passed by the user is originally a dictionary, but since `@lru_cache` wrapper only
-    accepts hashable parameters, the `"mongodb_extra_args"` should be converted into a frozenset beforehand.
+    accepts hashable arguments, the `"mongodb_extra_args"` should be converted into a frozenset beforehand.
```
Comment (jrobinAV) [inline taipy/core/common/_mongo_connector.py]:
```suggestion
    accepts hashable parameters, the `"mongodb_extra_args"` should be converted into a frozenset beforehand.
```

File: taipy/core/config/data_node_config.py [hunk 9]
```diff
@@ -610,20 +610,20 @@ def _configure_parquet(
...
+        Arguments:
             id (str): The unique identifier of the new Parquet data node configuration.
             default_path (Optional[str]): The default path of the Parquet file.
             engine (Optional[str]): Parquet library to use. Possible values are *"fastparquet"* or
                 *"pyarrow"*.<br/>
                 The default value is *"pyarrow"*.
             compression (Optional[str]): Name of the compression to use. Possible values are *"snappy"*,
                 *"gzip"*, *"brotli"*, or *"none"* (no compression). The default value is *"snappy"*.
-            read_kwargs (Optional[dict]): Additional parameters passed to the `pandas.read_parquet()`
+            read_kwargs (Optional[dict]): Additional arguments passed to the `pandas.read_parquet()`
```
Comment (jrobinAV) [inline taipy/core/config/data_node_config.py]:
```suggestion
            read_kwargs (Optional[dict]): Additional parameters passed to the `pandas.read_parquet()`
```

File: taipy/core/config/data_node_config.py [hunk 10]
```diff
@@ -610,20 +610,20 @@ def _configure_parquet(
...
                 function.
-            write_kwargs (Optional[dict]): Additional parameters passed to the
+            write_kwargs (Optional[dict]): Additional arguments passed to the
```
Comment (jrobinAV) [inline taipy/core/config/data_node_config.py]:
```suggestion
            write_kwargs (Optional[dict]): Additional parameters passed to the
```

File: taipy/core/config/data_node_config.py [hunk 11]
```diff
@@ -610,20 +610,20 @@ def _configure_parquet(
...
                 `pandas.DataFrame.write_parquet()` function.<br/>
-                The parameters in *read_kwargs* and *write_kwargs* have a **higher precedence** than the
-                top-level parameters which are also passed to Pandas.
+                The arguments in *read_kwargs* and *write_kwargs* have a **higher precedence** than the
+                top-level arguments which are also passed to Pandas.
```
Comment (jrobinAV) [inline taipy/core/config/data_node_config.py]:
```suggestion
                The parameters in *read_kwargs* and *write_kwargs* have a **higher precedence** than the
                top-level parameters which are also passed to Pandas.
```

File: taipy/core/config/data_node_config.py [hunk 12]
```diff
@@ -758,12 +758,12 @@ def _configure_in_memory(
...
         """Configure a new *in-memory* data node configuration.
 
-        Parameters:
+        Arguments:
             id (str): The unique identifier of the new in_memory data node configuration.
             default_data (Optional[any]): The default data of the data nodes instantiated from
                 this in_memory data node configuration.
                 If provided, note that the default_data will be stored as a configuration attribute.
-                So it is designed to handle small data values like parameters, and it must be Json serializable.
+                So it is designed to handle small data values like arguments, and it must be Json serializable.
```
Comment (jrobinAV) [inline taipy/core/config/data_node_config.py]:
```suggestion
                So it is designed to handle small data values like parameters, and it must be Json serializable.
```

File: taipy/core/config/data_node_config.py [hunk 13]
```diff
@@ -794,13 +794,13 @@ def _configure_pickle(
...
 
-        Parameters:
+        Arguments:
             id (str): The unique identifier of the new pickle data node configuration.
             default_path (Optional[str]): The path of the pickle file.
             default_data (Optional[any]): The default data of the data nodes instantiated from
                 this pickle data node configuration.
                 If provided, note that the default_data will be stored as a configuration attribute.
-                So it is designed to handle small data values like parameters, and it must be Json serializable.
+                So it is designed to handle small data values like arguments, and it must be Json serializable.
```
Comment (jrobinAV) [inline taipy/core/config/data_node_config.py]:
```suggestion
                So it is designed to handle small data values like parameters, and it must be Json serializable.
```

File: taipy/core/data/parquet.py [hunk 14]
```diff
@@ -45,14 +45,14 @@ class ParquetDataNode(DataNode, _FileDataNodeMixin, _TabularDataNodeMixin):
     - *compression* (`Optional[str]`): Name of the compression to use. Possible values
         are *"snappy"*, *"gzip"*, *"brotli"*, or *"none"* (no compression).<br/>
         The default value is *"snappy"*.
-    - *read_kwargs* (`Optional[dict]`): Additional parameters passed to the
+    - *read_kwargs* (`Optional[dict]`): Additional arguments passed to the
```
Comment (jrobinAV) [inline taipy/core/data/parquet.py]:
```suggestion
    - *read_kwargs* (`Optional[dict]`): Additional parameters passed to the
```

File: taipy/core/data/parquet.py [hunk 15]
```diff
@@ -45,14 +45,14 @@ class ParquetDataNode(DataNode, _FileDataNodeMixin, _TabularDataNodeMixin):
...
         *pandas.read_parquet()* function when reading the data.<br/>
-        The parameters in *"read_kwargs"* have a **higher precedence** than the top-level
-        parameters which are also passed to Pandas.
-    - *write_kwargs* (`Optional[dict]`): Additional parameters passed to the
+        The arguments in *"read_kwargs"* have a **higher precedence** than the top-level
+        arguments which are also passed to Pandas.
+    - *write_kwargs* (`Optional[dict]`): Additional Arguments passed to the
```
Comment (jrobinAV) [inline taipy/core/data/parquet.py]:
```suggestion
        The parameters in *"read_kwargs"* have a **higher precedence** than the top-level
        parameters which are also passed to Pandas.
    - *write_kwargs* (`Optional[dict]`): Additional parameters passed to the
```

File: taipy/core/data/parquet.py [hunk 16]
```diff
@@ -45,14 +45,14 @@ class ParquetDataNode(DataNode, _FileDataNodeMixin, _TabularDataNodeMixin):
...
         *pandas.DataFrame.write_parquet()* function when writing the data. <br/>
-        The parameters in *"write_kwargs"* have a **higher precedence** than the
-        top-level parameters which are also passed to Pandas.
+        The arguments in *"write_kwargs"* have a **higher precedence** than the
+        top-level arguments which are also passed to Pandas.
```
Comment (jrobinAV) [inline taipy/core/data/parquet.py]:
```suggestion
        The parameters in *"write_kwargs"* have a **higher precedence** than the
        top-level parameters which are also passed to Pandas.
```

File: taipy/core/scenario/scenario.py [hunk 17]
```diff
@@ -358,10 +358,10 @@ def subscribe(self, callback: Callable[[Scenario, Job], None], params: Optional[
         Note:
             Notification will be available only for jobs created after this subscription.
 
-        Parameters:
+        Arguments:
             callback (Callable[[Scenario^, Job^], None]): The callable function to be called
                 on status change.
-            params (Optional[List[Any]]): The parameters to be passed to the _callback_.
+            params (Optional[List[Any]]): The arguments to be passed to the _callback_.
```
Comment (jrobinAV) [inline taipy/core/scenario/scenario.py]:
```suggestion
            params (Optional[List[Any]]): The parameters to be passed to the _callback_.
```

File: taipy/core/scenario/scenario.py [hunk 18]
```diff
@@ -373,9 +373,9 @@ def unsubscribe(self, callback: Callable[[Scenario, Job], None], params: Optiona
         Note:
             The function will continue to be called for ongoing jobs.
 
-        Parameters:
+        Arguments:
             callback (Callable[[Scenario^, Job^], None]): The callable function to unsubscribe.
-            params (Optional[List[Any]]): The parameters to be passed to the _callback_.
+            params (Optional[List[Any]]): The arguments to be passed to the _callback_.
```
Comment (jrobinAV) [inline taipy/core/scenario/scenario.py]:
```suggestion
            params (Optional[List[Any]]): The parameters to be passed to the _callback_.
```

File: taipy/core/sequence/sequence.py [hunk 19]
```diff
@@ -264,10 +264,10 @@ def subscribe(
         Note:
             Notification will be available only for jobs created after this subscription.
 
-        Parameters:
+        Arguments:
             callback (Callable[[Sequence^, Job^], None]): The callable function to be called on
                 status change.
-            params (Optional[List[Any]]): The parameters to be passed to the _callback_.
+            params (Optional[List[Any]]): The arguments to be passed to the _callback_.
```
Comment (jrobinAV) [inline taipy/core/sequence/sequence.py]:
```suggestion
            params (Optional[List[Any]]): The parameters to be passed to the _callback_.
```

File: taipy/core/sequence/sequence.py [hunk 20]
```diff
@@ -279,9 +279,9 @@ def unsubscribe(self, callback: Callable[[Sequence, Job], None], params: Optiona
         Note:
             The function will continue to be called for ongoing jobs.
 
-        Parameters:
+        Arguments:
             callback (Callable[[Sequence^, Job^], None]): The callable function to unsubscribe.
-            params (Optional[List[Any]]): The parameters to be passed to the _callback_.
+            params (Optional[List[Any]]): The arguments to be passed to the _callback_.
```
Comment (jrobinAV) [inline taipy/core/sequence/sequence.py]:
```suggestion
            params (Optional[List[Any]]): The parameters to be passed to the _callback_.
```

File: taipy/core/taipy.py [hunk 21]
```diff
@@ -685,10 +685,10 @@ def subscribe_scenario(
     The subscription is applied to all jobs created for the execution of _scenario_.
     If no scenario is provided, the subscription applies to all scenarios.
 
-    Parameters:
+    Arguments:
         callback (Callable[[Scenario^, Job^], None]): The function to be called on
             status change.
-        params (Optional[List[Any]]): The parameters to be passed to the _callback_.
+        params (Optional[List[Any]]): The arguments to be passed to the _callback_.
```
Comment (jrobinAV) [inline taipy/core/taipy.py]:
```suggestion
        params (Optional[List[Any]]): The parameters to be passed to the _callback_.
```

File: taipy/core/taipy.py [hunk 22]
```diff
@@ -706,9 +706,9 @@ def unsubscribe_scenario(
 
     If no scenario is provided, the subscription is removed for all scenarios.
 
-    Parameters:
+    Arguments:
         callback (Callable[[Scenario^, Job^], None]): The function to unsubscribe from.
-        params (Optional[List[Any]]): The parameters to be passed to the callback.
+        params (Optional[List[Any]]): The arguments to be passed to the callback.
```
Comment (jrobinAV) [inline taipy/core/taipy.py]:
```suggestion
        params (Optional[List[Any]]): The parameters to be passed to the callback.
```

File: taipy/core/taipy.py [hunk 23]
```diff
@@ -725,10 +725,10 @@ def subscribe_sequence(
 
     The subscription is applied to all jobs created for the execution of _sequence_.
 
-    Parameters:
+    Arguments:
         callback (Callable[[Sequence^, Job^], None]): The callable function to be called on
             status change.
-        params (Optional[List[Any]]): The parameters to be passed to the _callback_.
+        params (Optional[List[Any]]): The arguments to be passed to the _callback_.
```
Comment (jrobinAV) [inline taipy/core/taipy.py]:
```suggestion
        params (Optional[List[Any]]): The parameters to be passed to the _callback_.
```

File: taipy/core/taipy.py [hunk 24]
```diff
@@ -743,10 +743,10 @@ def unsubscribe_sequence(
 ) -> None:
     """Unsubscribe a function that is called when the status of a Job changes.
 
-    Parameters:
+    Arguments:
         callback (Callable[[Sequence^, Job^], None]): The callable function to be called on
             status change.
-        params (Optional[List[Any]]): The parameters to be passed to the _callback_.
+        params (Optional[List[Any]]): The arguments to be passed to the _callback_.
```
Comment (jrobinAV) [inline taipy/core/taipy.py]:
```suggestion
        params (Optional[List[Any]]): The parameters to be passed to the _callback_.
```

File: taipy/core/task/task.py [hunk 25]
```diff
@@ -28,7 +28,7 @@
 
 
 class Task(_Entity, _Labeled):
-    """Hold a user function that will be executed, its parameters and the results.
+    """Hold a user function that will be executed, its arguments and the results.
```
Comment (jrobinAV) [inline taipy/core/task/task.py]:
```suggestion
    """Hold a user function that will be executed, its parameters, and the results.
```

File: taipy/gui/gui.py [hunk 26]
```diff
@@ -308,7 +308,7 @@ def __init__(
 
         - *state*: the `State^` instance of the caller.
         - *page_name*: the name of the page the user is navigating to.
-        - *params* (Optional): the query parameters provided in the URL.
+        - *params* (Optional): the query arguments provided in the URL.
```
Comment (jrobinAV) [inline taipy/gui/gui.py]:
```suggestion
        - *params* (Optional): the query parameters provided in the URL.
```

File: taipy/gui/gui.py [hunk 27]
```diff
@@ -352,10 +352,10 @@ def __init__(
 
         - *state*: the `State^` instance of the caller.
         - *path*: the path provided to the `get_user_content_url()^` to build the URL.
-        - *parameters*: An optional dictionary as defined in the `get_user_content_url()^` call.
+        - *arguments*: An optional dictionary as defined in the `get_user_content_url()^` call.
```
Comment (jrobinAV) [inline taipy/gui/gui.py]:
```suggestion
        - *parameters*: An optional dictionary as defined in the `get_user_content_url()^` call.
```

File: taipy/gui/gui.py [hunk 28]
```diff
@@ -352,10 +352,10 @@ def __init__(
...
 
         The returned HTML content can therefore use both the variables stored in the *state*
-        and the parameters provided in the call to `get_user_content_url()^`.
+        and the arguments provided in the call to `get_user_content_url()^`.
```
Comment (jrobinAV) [inline taipy/gui/gui.py]:
```suggestion
        and the parameters provided in the call to `get_user_content_url()^`.
```

File: taipy/gui/gui.py [hunk 29]
```diff
@@ -1604,8 +1604,8 @@ def broadcast_callback(
             callback: The user-defined function to be invoked.<br/>
                 The first parameter of this function must be a `State^` object representing the
                 client for which it is invoked.<br/>
-                The other parameters should reflect the ones provided in the *args* collection.
-            args: The parameters to send to *callback*, if any.
+                The other arguments should reflect the ones provided in the *args* collection.
+            args: The arguments to send to *callback*, if any.
```
Comment (jrobinAV) [inline taipy/gui/gui.py]:
```suggestion
                The other parameters should reflect the ones provided in the *args* collection.
            args: The parameters to send to *callback*, if any.
```

File: taipy/gui/gui_actions.py [hunk 30]
```diff
@@ -44,7 +44,7 @@ def download(
         - *on_action*: this callback is triggered when the transfer of the content is achieved.</br>
           In this function, you can perform any clean-up operation that could be required after
           the download is completed.<br/>
-          This callback can use three optional parameters:
+          This callback can use three optional arguments:
```
Comment (jrobinAV) [inline taipy/gui/gui_actions.py]:
```suggestion
          This callback can use three optional parameters:
```

File: taipy/gui/gui_actions.py [hunk 31]
```diff
@@ -165,7 +165,7 @@ def navigate(
         to: The name of the page to navigate to. This can be a page identifier (as created by
             `Gui.add_page()^` with no leading '/') or a URL.<br/>
             If omitted, the application navigates to the root page.
-        params: A dictionary of query parameters.
+        params: A dictionary of query arguments.
```
Comment (jrobinAV) [inline taipy/gui/gui_actions.py]:
```suggestion
        params: A dictionary of query parameters.
```

File: taipy/gui/gui_actions.py [hunk 32]
```diff
@@ -190,7 +190,7 @@ def get_user_content_url(
         state (State^): The current user state as received in any callback.
         path: An optional additional path to the URL.
         params: An optional dictionary sent to the *on_user_content* callback.<br/>
-            These arguments are added as query parameters to the generated URL and converted into
+            These arguments are added as query arguments to the generated URL and converted into
```
Comment (jrobinAV) [inline taipy/gui/gui_actions.py]:
```suggestion
            These arguments are added as query parameters to the generated URL and converted into
```

File: taipy/gui/gui_actions.py [hunk 33]
```diff
@@ -308,8 +308,8 @@ def broadcast_callback(
         callback: The user-defined function to be invoked.<br/>
             The first parameter of this function must be a `State^` object representing the
             client for which it is invoked.<br/>
-            The other parameters should reflect the ones provided in the *args* collection.
-        args: The parameters to send to *callback*, if any.
+            The other arguments should reflect the ones provided in the *args* collection.
+        args: The arguments to send to *callback*, if any.
```
Comment (jrobinAV) [inline taipy/gui/gui_actions.py]:
```suggestion
            The other parameters should reflect the ones provided in the *args* collection.
        args: The parameters to send to *callback*, if any.
```

File: taipy/gui/server.py [hunk 34]
```diff
@@ -357,7 +357,7 @@ def run(
         }
         if self.__ssl_context is not None:
             run_config["ssl_context"] = self.__ssl_context
-        # flask-socketio specific conditions for 'allow_unsafe_werkzeug' parameters to be popped out of kwargs
+        # flask-socketio specific conditions for 'allow_unsafe_werkzeug' arguments to be popped out of kwargs
```
Comment (jrobinAV) [inline taipy/gui/server.py]:
```suggestion
        # flask-socketio specific conditions for 'allow_unsafe_werkzeug' parameters to be popped out of kwargs
```

File: taipy/rest/rest.py [hunk 35]
```diff
@@ -24,13 +24,13 @@ def __init__(self) -> None:
         """
         Initialize a REST API server.
 
-        A Flask application is instantiated and configured using three parameters from the global
+        A Flask application is instantiated and configured using three arguments from the global
```
Comment (jrobinAV) [inline taipy/rest/rest.py]:
```suggestion
        A Flask application is instantiated and configured using three parameters from the global
```

File: taipy/rest/rest.py [hunk 36]
```diff
@@ -24,13 +24,13 @@ def __init__(self) -> None:
...
         config.
             - Config.global_config.testing (bool): Run the application on testing mode.
             - Config.global_config.env (Optional[str]): The application environment.
             - Config.global_config.secret_key (Optional[str]): Application server secret key.
 
-        However, editing these parameters is only recommended for advanced users. Indeed, the default behavior of the
+        However, editing these arguments is only recommended for advanced users. Indeed, the default behavior of the
```
Comment (jrobinAV) [inline taipy/rest/rest.py]:
```suggestion
        However, editing these parameters is only recommended for advanced users. Indeed, the default behavior of the
```

**Review by jrobinAV — DISMISSED** [review]

File: taipy/common/config/config.pyi [hunk 37]
```diff
@@ -508,8 +508,8 @@ class Config:
                 function.
             write_kwargs (Optional[dict]): Additional parameters passed to the
                 `pandas.DataFrame.write_parquet()` function.<br/>
-                The parameters in *read_kwargs* and *write_kwargs* have a **higher precedence** than the
-                top-level parameters which are also passed to Pandas.
+                The parameters in *read_kwargs* and *write_kwargs* have a **higher precedence**
```
Comment (jrobinAV) [inline taipy/common/config/config.pyi]:
```suggestion
                The parameters in *read_kwargs* and *write_kwargs* have a **higher precedence**
```

**Comment by jrobinAV** [comment]
Thank you for your contribution @DevyManish.

**Review by jrobinAV — APPROVED** [review]

**Comment by RymMichaut** [comment]
Congratulations, @DevyManish , for this merged Hacktoberfest PR.
Can you please contact me at rym.michaut@taipy.io to prepare your swag and send it to you ASAP?

**Comment by DevyManish** [comment]
Sorry i was little busy with my exams ... what details i need to provide

**Comment by DevyManish** [comment]
@RymMichaut  please check my email sent from manishguptafr@gmail.com[](manishguptafr@gmail.com)

**Comment by RymMichaut** [comment]
Hello,

Could you please provide your full name, full postal address and phone
number?

Thanks

Regards,
------
*PS*: I don't work on Wednesdays

<https://github.com/Avaiga/taipy/issues?q=is%3Aopen+is%3Aissue+label%3Ahacktoberfest>
[image:
https://github.com/Avaiga/taipy/issues?q=is%3Aopen+is%3Aissue+label%3Ahacktoberfest]
<https://github.com/Avaiga/taipy/issues?q=is%3Aopen+is%3Aissue+label%3Ahacktoberfest>

<https://www.linkedin.com/in/rymguerbi/> <https://github.com/RymMichaut>



Rym Michaut

🧢*Head of Digital Marketing & Communcation*


📧 ***@***.***


📱+33 6 72 40 28 44


Le jeu. 24 oct. 2024 à 21:02, Manish Gupta ***@***.***> a
écrit :

**Pull request merged**